### ISSUE TITLE
[Question]: Hello, I want to split the back-end code of upload document and read parsed document in RAGFLOW, I don't need WEB page. Please tell me where I should start.

### ISSUE THREAD

**Issue by cheun726** [infiniflow/ragflow#2164]
### Describe your problem

Hello, I want to split the back-end code of upload document and read parsed document in RAGFLOW, I don't need WEB page. Please tell me where I should start.As clear as possible.Thanks

**Comment by saineshwar** [comment]
You can use API of Ragflow 
[API URL Link ](https://ragflow.io/docs/dev/api)

**Comment by cheun726** [comment]
I don’t wanna the api.Please tell me how to locate the code of the corresponding function?

**Comment by cheun726** [comment]
@KevinHuSh  Please help me

**Comment by JinHai-CN** [comment]
Yes, you should use the API and the web UI can be considered as a GUI for administration.

**Comment by KevinHuSh** [comment]
> @KevinHuSh Please help me

There're will be python SDK released soon.
